Dynatrace/dynatrace-operator
# Adds v1alpha2 EdgeConnect CRD

[K8S-10077](https://dt-rnd.atlassian.net/browse/K8S-10077)

## Description

Adds v1alpha2 ~v1beta1~ EdgeConnect CRD.

## How can this be tested?

1) Install operator from the main branch.

2) Install EdgeConnect of version v1alpha1 with `hostRestrictions`:
```
apiVersion: dynatrace.com/v1alpha1
spec:
  hostRestrictions: "a.au,b.au" 
```

3) Update operator.

4) Check hostRestrictions:
```
apiVersion: dynatrace.com/v1alpha2
spec:
  hostRestrictions:
  - a.au
  - b.au
```
5) Check operator logs - no errors expected
